pyEDAA/UCIS/Cobertura.py

```diff
@@ -45,47 +45,83 @@
 from pyTooling.Decorators import export
 
 
+@export
+class CoberturaException(Exception):
+	"""Base class for other Cobertura exceptions"""
+	pass
```

Comment:
@por3bski The exception structure is very good.

In case of "empty" classes, a pass statement is not needed when a doc-string was provided.

Reply:
There was no "suggested change" for this issue.
Fixed in https://github.com/edaa-org/pyEDAA.UCIS/pull/7/commits/b61cd023dfa518c16cdef7e12f01642fa7046bd8

Reply:
Checked.